src/components/MessageListItem.react.js

```diff
@@ -45,6 +54,41 @@ class MessageListItem extends React.Component {
         let tableData = data.answers[0].data;
```

Comment:
@daminisatya I guess the highlighting is because its in ES6 and the editor needs a babel plugin to support it. The code works fine. Did you mean to fix that only?